ffi/driver.cc

```diff
@@ -36,6 +38,22 @@ void init_ffi_driver(py::module_ &m) {
         .def("sync", &Driver::sync)
         .def("collect_returns", &Driver::collectReturns)
         .def("time", &Driver::time, "rounds"_a = 10, "warmpups"_a = 3);
+
+    // Serialization
+    m.def("load_target", &loadTarget);
+    m.def("load_device", &loadDevice);
+    m.def("load_array", &loadArray);
+    m.def("dump_target", &dumpTarget, "target"_a);
+    m.def("dump_device", &dumpDevice, "device"_a);
+    m.def(
+        "dump_array",
+        [](const Ref<Array> &array_) {
+            auto &&[ret_meta, ret_data] = dumpArray(array_);
+            return std::make_pair(ret_meta, py::bytes(ret_data));
+        },
+        "array"_a);
+    m.def("new_test_array", &newArray, "shape"_a, "dtype"_a, "devices"_a,
+          "data"_a);
```

Comment:
You can directly construct an `Array` from a NumPy array. No need to add a new interface. For example, https://github.com/roastduck/FreeTensor/blob/master/test/00.hello_world/test_basic.py#L146